snap/local/hooks/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/edgexfoundry/app-rfid-llrp-inventory/hooks
```

Comment:
missing `go.sum` file for this `go.mod`

Reply:
I ran a go mod tidy, go.sum file is there now